### PR TITLE
[CallStack] More test coverage for framework grouping

### PR DESCRIPTION
**Pull request by calebstdenis**
Associated Issue: #5150 

### Summary of Changes
* Increased test coverage of existing framework grouping functionality before I change it. More specifically, I added a snapshot test for the Webpack edge case.
* Little change to vscode config for peace of mind

image

### CODE REVIEW

File: src/components/SecondaryPanes/Frames/tests/Frames.spec.js [hunk 1]
```diff
@@ -122,5 +122,32 @@ describe("Frames", () => {
...
+        {
+          id: "5-webpackBundleFrame",
+          source: { url: "https://foo.com/bundle.js" }
+        }
+      ];
+      const selectedFrame = frames[0];
+      const frameworkGroupingOn = true;
+      const component = render({ frames, frameworkGroupingOn, selectedFrame });
+
+      expect(component).toMatchSnapshot();
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/Frames/tests/Frames.spec.js]: nice!

I'm surprised this worked without the connect function
